demo/docker/testData/transactions-value.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
what aboud alerts-value? Can we add schemas automatically

Reply:
I've created files with Schema Registry definition for all of the subjects and added script to load them into the Schema Registry